Cargo.lock

```diff
@@ -66,6 +66,12 @@ dependencies = [
  "serde",
 ]
 
+[[package]]
+name = "android-tzdata"
```

Comment:
This doesn't seem very useful for us :) But it's enabled by default by `chrono`. We'd need to manually enable features to get rid of it.

Reply:
It's from clock feature from `chrono` https://github.com/chronotope/chrono/blob/v0.4.38/Cargo.toml#L26, which is selected by `postgres-types` https://github.com/sfackler/rust-postgres/blob/647a9256b80ef961b8065678f41745ca040202b5/postgres-types/Cargo.toml#L39-L41